File: csv/clojure/read/csv_read.clj
; -----------------------------------------------------------------
;
;	csv_read.clj
;
;					Oct/27/2011
;
; -----------------------------------------------------------------
(load-file "/var/www/data_base/common/clojure_common/text_manipulate.clj")
; -----------------------------------------------------------------
(println "*** 開始 ***")
(def file_text (first *command-line-args*))
;
(def dict_aa (csv_read_proc file_text))

(println (class dict_aa))
;
(dict_display_proc dict_aa)
(println "*** 終了 ***")
; ----------------------------------------------------------------

File: csv/dart/read/csv_read.dart
/*
	csv_read.dart

					Jan/10/2021
*/
// -------------------------------------------------------------
import 'dart:io'; 

import	'/var/www/data_base/common/dart_common/text_manipulate.dart';
// -------------------------------------------------------------
main (List<String> arguments) {
	stderr.writeln ("*** 開始 ***");

	String file_name = '${arguments[0]}';

	File file = new File (file_name);
	Future future = file.readAsString ();

	future.then((content)
		{
		var dict_aa = str_to_dict_proc (content,',');

		dict_display_proc (dict_aa);

		stderr.writeln ("*** 終了 ***");
		}); 
}

// -------------------------------------------------------------
